### PR TITLE
feat: updated docs descriptions 

### LINKED ISSUE #7606: [Docs] Update Naming Convention
#### Current State
Usage of Service Mesh at several instances can be found 
image

#### Desired State
Replace `Service Mesh` with `Cloud Native` at [ reference/mesheryctl](https://docs.meshery.io/reference/mesheryctl) and its related components

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by thisiskaransgit**
**Notes for Reviewers**

This PR fixes #7606 

Updated the Description and replaced `Service Mesh` with `Cloud Native` at [reference/mesheryctl](https://docs.meshery.io/reference/mesheryctl)

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 41e409ac371108a5a3b90d2a219c6c8fcd1f89ba at: https://64483f7f9de61f220c659499--meshery-docs-preview.netlify.app

File: docs/pages/getting-started/overview.md [hunk 1]
```diff
@@ -11,18 +11,18 @@ As the cloud native management plane, Meshery enables the adoption, operation, a
...
    - Workload and service mesh performance characterization
    - Prometheus and Grafana integration
-1. Service Mesh Configuration Management
+1. Cloud Native Configuration Management
    - Configuration best practices
-1. Service Mesh Lifecycle Management
+1. Cloud Native Lifecycle Management
    - Service mesh provisioning and workload onboarding
    - Meshery Operator and MeshSync
    - Service mesh patterns and Open Application Model integration
```
Comment (leecalcote) [inline docs/pages/getting-started/overview.md:22]: We're missing a couple of references that can be swapped out here...

File: docs/pages/reference/mesheryctl-commands.md [hunk 2]
```diff
@@ -17,10 +17,10 @@ Meshery CLI commands are categorized by function, which are:
 
 - `mesheryctl` - Global flags and CLI configuration
 - `mesheryctl system` - Meshery Lifecycle and Troubleshooting
-- `mesheryctl mesh` - Service Mesh Lifecycle & Configuration Management: provisioning and configuration best practices
-- `mesheryctl perf` - Service Mesh Performance Management: Workload and service mesh performance characterization
-- `mesheryctl pattern` - Service Mesh Pattern Configuration & Management: Service mesh patterns and Open Application Model integration
-- `mesheryctl app` - Service Mesh Application Management
+- `mesheryctl mesh` - Cloud Native Lifecycle & Configuration Management: provisioning and configuration best practices
+- `mesheryctl perf` - Cloud Native Performance Management: Workload and service mesh performance characterization
+- `mesheryctl pattern` - Cloud Native Pattern Configuration & Management: Service mesh patterns and Open Application Model integration
```
Comment (leecalcote) [inline docs/pages/reference/mesheryctl-commands.md:22]: Missed a reference here.

Comment (thisiskaransgit) [reply docs/pages/reference/mesheryctl-commands.md:22]: Thank you for pointing out, resolved it!

File: docs/pages/getting-started/overview.md (hunk 1 above)
Comment (thisiskaransgit) [reply docs/pages/getting-started/overview.md]: Those were sub-points, hence I left them earlier, have updated them now.

**Comment by l5io** [comment]
🚀 Preview for commit 967b520ac172caeb6be794fc10a8cb505b7a98a5 at: https://6448c1ec30ba041a77a61ad6--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit fde862784d2fae211381d8511990a85a856b5d00 at: https://6450f2dc5d09184b91e087f0--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]
Great work, @thisiskaransgit 👏

**Pull request merged**